.gitignore

```diff
@@ -95,3 +95,6 @@ cython_debug/
 *.iws
 out/
 
+### pyenv
+.python-version
+nohup.out
```

Comment:
Ok to have that, but definitely some interesting techniques if yo find yourself having nohup ;)